modules/Bio/EnsEMBL/IO/Parser/BaseVCF4.pm

```diff
@@ -316,11 +316,10 @@ sub get_end {
     elsif(defined($info->{SVLEN})) {
```

Comment:
In VCF specification 4.4 `SVLEN` is given precedence over `END` for symbolic SVs. See -
this discussion, https://github.com/samtools/hts-specs/issues/647
and, section 3 in the specification, https://samtools.github.io/hts-specs/VCFv4.4.pdf

So, we should first try to calculate end position from `SVLEN`. If there is no `SVLEN` we can try to get it from `END` if available.

Also according to specification, the calculation for `SVLEN` would be - `end = POS + SVLEN` which interprets in our code - `$end = $self->get_raw_start + abs($svlen);` but the current calculation is also creates same result. We can change it if we want to make the code more interpretable with the specs.

Fixes the issue https://github.com/Ensembl/ensembl-vep/issues/1765